math/src/field/fields/u64_prime_field.rs

```diff
@@ -86,15 +104,10 @@ impl<const ORDER: u64> IsCyclicBilinearGroup for U64FieldElement<ORDER> {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use test_case::test_case;
     const ORDER: u64 = 13;
     type FE = FieldElement<U64PrimeField<ORDER>>;
 
-    #[test]
-    fn order_must_small_as_to_not_allow_overflows() {
-        // ORDER*ORDER < u128::MAX
-        assert!(ORDER <= u64::MAX.into());
-    }
```

Comment:
This had a warning from clippy that said that it made no sense. Mostly because `ORDER` is a `u64` so clearly `<= u64::MAX`. Maybe the original intention of this test was another one?